app/src/main/java/com/bottlerocketstudios/mapsdemo/ui/map/YelpViewModel.kt

```diff
@@ -17,26 +21,70 @@ class YelpViewModel : BaseViewModel() {
 
     // UI
     val yelpBusinessState: MutableStateFlow<List<Business>> = MutableStateFlow(emptyList())
-    val dallasLatLng: LatLng = LatLng(LATITUDE, LONGITUDE)
+    val googleMapsMarkersLatLng = yelpBusinessState.map { businessList ->
+        businessList.map { business ->
+            YelpMarker(
+                latitude = business.coordinates.latitude,
+                longitude = business.coordinates.longitude,
+                businessName = business.businessName
+            )
+        }
+    }
+    val selectedMarker: MutableStateFlow<YelpMarker> = MutableStateFlow(YelpMarker(latitude = 0.0, longitude = 0.0, businessName = ""))
+    val dallasLatLng: LatLng = LatLng(DEFAULT_LATITUDE, DEFAULT_LONGITUDE)
+    private var searchJob: Job? = null
 
     private companion object {
-        const val LATITUDE = 32.7767
-        const val LONGITUDE = -96.7970
+        const val DEFAULT_LATITUDE = 32.7767
+        const val DEFAULT_LONGITUDE = -96.7970
+        const val SEARCH_DELAY = 300L
+        const val ZOOM_THRESHOLD = 13
```

Comment:
@Josh-Gebbeken Since zoomLevel in the `getYelpBusinessesOnMapMove` function takes in a float, should this value be defined as a Float as well? If so, change to:

```suggestion
        const val ZOOM_THRESHOLD = 13f
```